packages/frontend/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@apollo/client": "^3.2.2",
     "@lara/components": "^1.0.0",
+    "@react-oauth/google": "^0.5.1",
```

Comment:
the old react-google-login can be removed than. Right?

Reply:
fixed